src/wallet/rpcwallet.cpp

```diff
@@ -967,10 +969,11 @@ static UniValue addmultisigaddress(const JSONRPCRequest& request)
                     {"address_type", RPCArg::Type::STR, /* default */ "set by -addresstype", "The address type to use. Options are \"legacy\", \"p2sh-segwit\", and \"bech32\"."},
                 },
                 RPCResult{
-            "{\n"
-            "  \"address\":\"multisigaddress\",    (string) The value of the new multisig address.\n"
-            "  \"redeemScript\":\"script\"         (string) The string value of the hex-encoded redemption script.\n"
-            "}\n"
+                    RPCResult::Type::OBJ, "", "",
+                    {
+                        {RPCResult::Type::STR, "address", "The value of the new multisig address"},
+                        {RPCResult::Type::STR, "redeemScript", "The string value of the hex-encoded redemption script"},
```

Comment:
can this be of type HEX_STR? The comment/description mentions `hex-econded`

Reply:
Fixed in 2fe4fda6ec